litmus-portal/Makefile

```diff
@@ -7,18 +7,6 @@
 #
 IS_DOCKER_INSTALLED = $(shell which docker >> /dev/null 2>&1; echo $$?)
```

Comment:
We should keep the makefile as it is, it helps in checking and building the services locally.